src/materials/MeshPhysicalMaterial.js

```diff
@@ -40,7 +40,7 @@ class MeshPhysicalMaterial extends MeshStandardMaterial {
 		} );
 
 		this.iridescenceMap = null;
-		this.iridescenceIOR = 1.3;
```

Comment:
@PascalSchoen sorry for the churn here, but glTF's camelCase convention for acronyms like XYZ (`fooBarXyz`) is different from what we follow in three.js (`fooBarXYZ`). I think for the public API of MeshPhysicalMaterial, this should remain `iridescenceIOR`. Feel free to use whichever you prefer for internal code in the rest of the PR. 🙏

Reply:
I think so, yes — for example we use:

- attribute.getXYZ not .getXyz
- attribute.x not attribute.X
- GLTFLoader not GltfLoader

I'm comfortable with either convention but keeping acronyms uppercase seems to be the norm in three.js today, and (I think?) more common in JavaScript generally.

Reply:
Slight preference for applyIORToRoughness, yes - but not as worried about internal methods.

Somehow `.ior` does not bother me, feels similar to `vec.xyz` in GLSL. 🤷‍♂️